geometric_features/aggregation/ocean/arctic_regions.py

```diff
@@ -0,0 +1,54 @@
+from geometric_features import FeatureCollection
+
+
+def arctic(gf):
+    """
+    Aggregate Arctic regions
+
+    Parameters
+    ----------
+    gf : geometric_features.GeometricFeatures
+        An object that knows how to download and read geometric features
+
+    Returns
+    -------
+    fc : geometric_features.FeatureCollection
+        The new feature collection with Arctic regions
+    """
+    # Authors
+    # -------
+    # Milena Veneziani, Xylar Asay-Davis
+
+    regions = {'Baffin Bay': ['Baffin Bay'],
+               'Baltic Sea': ['Skaggerak', 'Kattegat', 'Gulf of Bothnia',
+                              'Gulf of Finland', 'Baltic Sea', 'Gulf of Riga'],
+               'Barents Sea': ['Barents Sea', 'White Sea'],
+               'Canada Basin': ['Canada Basin'],
+               'Canadian Archipelago': ['Canadian Archipelago'],
+               'Central Arctic': ['Central Arctic'],
+               'Chukchi Sea': ['Chukchi Sea'],
+               'East Siberian Sea': ['East Siberian Sea'],
+               'Greenland Sea': ['Greenland Sea'],
+               'Hudson Bay': ['Hudson Bay'],
+               'Irminger Sea': ['Irminger Sea'],
+               'Kara Sea': ['Kara Sea'],
+               'Labrador Sea': ['Labrador Sea'],
+               'Laptev Sea': ['Laptev Sea'],
+               'North Sea': ['North Sea'],
+               'Norwegian Sea': ['Norwegian Sea'],
+               'Beaufort Gyre': ['Beaufort Gyre'],
+               'Beaufort Gyre Shelf': ['Beaufort Gyre Shelf'],
+               'Arctic Ocean without Barents and Kara Seas': [
```

Comment:
@milenaveneziani, the previous name of this feature (`Arctic Ocean (no Barents/Kara Seas)`) causes a lot of trouble in MPAS-Analysis. It seems like we need to avoid punctuation like parentheses and slashes in feature names.